### ISSUE TITLE
Training vargfacenet model produces nan's

### ISSUE THREAD

**Issue by yossibiton** [deepinsight/insightface#1295]
I started to train y2 model on retina dataset and it works fine (loss it not NaN) : 
`CUDA_VISIBLE_DEVICES='0,1,2,3' python -u train.py --dataset retina --network y2 --loss arcface`

When i switch to vargfacenet network the loss get NaN value right away in the first batch : 
`CUDA_VISIBLE_DEVICES='0,1,2,3' python -u train.py --dataset retina --network vargfacenet --loss arcface`
I thought this is because of the batch size so i tried other values for per-batch-size (64, 32) but it still fails.

Can someone explain that ?

**Comment by tranvanhoa533** [comment]
Have you solved this problem?

**Comment by yossibiton** [comment]
No, I haven't.
I don't use this backbone for now.

**Comment by tk4218** [comment]
This issue was addressed in #980. Reduce your initial learning rate to 0.01. 

Also, I have found that larger batch sizes (> 128) produce the same issue after a while, so if you still run into the issue after adjusting the learning rate, try using a smaller batch size.
